dora/core/client/fs/src/main/java/alluxio/client/file/FileSystem.java

```diff
@@ -388,11 +389,17 @@ List<BlockLocationInfo> getBlockLocations(URIStatus status)
    * @return the {@link URIStatus} of the file
    * @throws FileDoesNotExistException if the path does not exist
    */
+  @Deprecated
   default URIStatus getStatus(AlluxioURI path)
       throws FileDoesNotExistException, IOException, AlluxioException {
     return getStatus(path, GetStatusPOptions.getDefaultInstance());
   }
 
+  default URIStatus getStatus(UfsUrl ufsPath)
```

Comment:
no matter whether the method is `default` or not, we just create one replacement using `UfsUrl`